### DISCUSSION TITLE
Create 3 Adapter board not working

### DISCUSSION THREAD

**Discussion by niebrandon** [iRobotEducation/create3_docs#237] [Questions]
Hi, my IROBOT CREATE 3 will not connect to bluetooth via the web sdk, and it is not being recognized by my raspi 4 in ros2 galactic over usb (it used to work perfectly) I checked and made sure the board was on the bluetooth switch when attempting to connect via bluetooth, and that it was on usb when attempting to perform a usb connection, I also have tried rebooting the robot, and disconnecting/reconecting the adapter board. All of the indicators on the board seem to be working fine as well, I am not sure what the problem is, and if there is a solution. If it is possible to fix, or buy a replacement adapter board please let me know.

**Comment by shamlian** [comment]
Don't panic! 

Could you flip the switch one way, wait 30 seconds, then flip the switch the other way, and then connect to the webserver over Wi-Fi and download the logs to post here? If you'd rather, you can also email them to education@irobot.com . 

Did something happen that might have caused the adapter board to break? Just looking for clues as to why you might need a new one. My understanding is that new adapter boards should be available to purchase soon, but I'm not sure when "soon" is. That said, I'm sure we can help you, one way or another.

**Reply by niebrandon** [reply]
[messages.txt](https://github.com/iRobotEducation/create3_docs/files/9986314/messages.txt)
here are the logs from my create 3 I flipped it from bluetooth to usb mode set a timer for 30 seconds, then flipped it back, so it may be a little longer than 30 seconds 
as far as I can remember nothing unusual has happened, our power has been flickering over the last few days, however both my create 3, and my Roomba are on the same surge protector and my Roomba experienced no damage (as far as I can tell) but besides from that there is nothing else that I can think of

**Reply by shamlian** [reply]
This log seems to just show a boot process, with no change on the adapter board at all, so I agree something seems wrong in hardware, either with the adapter board or the Bluetooth radio (or perhaps even with the robot itself, though that seems less likely). My next step would have been to ask you to plug in a device like a network adapter to see if it shows up in the logs, but it sounds like that's something that used to work and no longer does. At this point, I think you should email education@irobot.com referencing this ticket -- it's not my call but I suspect they will want to arrange an RMA for your adapter board.
